src/components/Review/ReviewSection.tsx

```diff
@@ -1,39 +1,58 @@
 import React from 'react'
-import { Button, Grid, Header, Icon, Segment } from 'semantic-ui-react'
+import { Button, Container, Grid, Header, Icon, Label, Segment } from 'semantic-ui-react'
 import { SummaryViewWrapper } from '../../formElementPlugins'
 import strings from '../../utils/constants'
 import { ReviewResponseDecision, TemplateElementCategory } from '../../utils/generated/graphql'
 import { ReviewQuestionDecision, ResponsesByCode, SectionElementStates } from '../../utils/types'
 
 interface ReviewSectionProps {
   allResponses: ResponsesByCode
+  assignedToYou: boolean
   reviewSection: SectionElementStates
   updateResponses: (props: ReviewQuestionDecision[]) => void
   canEdit: boolean
 }
 
 const ReviewSection: React.FC<ReviewSectionProps> = ({
   allResponses,
+  assignedToYou,
   reviewSection,
   updateResponses,
   canEdit,
 }) => {
   const { section, pages } = reviewSection
+
+  const getSectionAssignment = assignedToYou ? (
+    <Label style={{ backgroundColor: 'WhiteSmoke', color: 'Blue' }}>
+      {strings.LABEL_ASSIGNED_TO_YOU}
+    </Label>
+  ) : (
+    <Label style={{ backgroundColor: 'WhiteSmoke' }}>{strings.LABEL_ASSIGNED_TO_OTHER}</Label>
+  )
   return (
     <Segment
       key={`ReviewSection_${section.code}`}
       inverted
-      style={{ backgroundColor: 'WhiteSmoke', margin: '15px 50px 0px' }}
+      style={{ backgroundColor: 'WhiteSmoke', marginLeft: '10%', marginRight: '10%' }}
     >
-      <Header as="h2" content={`${section.title}`} style={{ color: 'Grey' }} />
+      <Grid columns={2}>
+        <Grid.Row>
+          <Grid.Column>
+            <Header as="h2" content={`${section.title}`} style={{ color: 'Grey' }} />
+          </Grid.Column>
+          <Grid.Column>
+            <Container textAlign="right">{getSectionAssignment}</Container>
```

Comment:
Just something to think about, maybe don't want to change it for now, but -- I kind of think of `get` methods as functions that return a value. I wonder if functions that return visual components (i.e. JSX) might be called `show..` e.g. `showSectionAssignment` (or maybe `displaySectionAssignment). Cos you're really _getting_ the assignment, you're getting the component to _display_ the assignment. Subtle point, see what you think.

I like the general pattern of functions that return JSX though, I'll try and think to use it more. :)

Reply:
No problem I'll change this one.